man/logind.conf.xml

```diff
@@ -200,6 +192,20 @@
         <xi:include href="version-info.xml" xpointer="v240"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>SleepOperations=</varname></term>
```

Comment:
I'd name these in singular, not plural. I mean, sure it takes a list of things, but we only execute one of them after all, and for most people that's what matters. Otherwise it really sounds as if all of them are used, but that's not the case.

In particular as people who actually configure this would typically configure a single one of them only, not really a list.